.coderabbit.yaml

```
@@ -16,11 +16,19 @@ reviews:
   # Request more detailed analysis for complex files
   profile: "assertive"
   
+  # Large codebase optimization - enable incremental review
+  incremental: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)